### ISSUE TITLE
iOS app: Support text size changes

### ISSUE THREAD

**Issue by likethegoddess** [mastodon/mastodon#24309]
### Pitch

Text size for the Mastodon app is not adjustable on iOS. Both the steps to reproduce below result in text size that does not change. 

### Steps 1
1. Open to Settings then Accessibility. 
2. Open Display and Text Size then select Larger Text. 
3. Switch Larger Accessibility Sizes on. 
4. Adjust text size with slider. 

### Steps 2
1. Open to Settings then Control Center. 
5. Add Text Size to your Included Controls if not already present. 
6. Launch Mastodon. 
7. Open Control Center and select the Text Size icon. 
8. Change text size for Mastodon only. 
9. Tap to close then swipe up

### Motivation

Users should be able to change text size as needed for accessibility.

**Comment by likethegoddess** [comment]
Wrong repo, moving issue.
